EazyE2E/Element/EzRootElement.cs

```diff
@@ -12,22 +12,22 @@ namespace EazyE2E.Element
 {
     public class EzRoot
     {
-        public EzElement RootElement;
+        private readonly EzProcess _process;
+        private EzElement _rootElement;
 
-        public EzRoot(string processName)
```

Comment:
Process name is now a parameter of the EzProcess object's constrcutor.  This will allow us to ensure we are using the same process throughout.